src/main/java/com/es/phoneshop/web/CheckoutPageServlet.java

```diff
@@ -0,0 +1,133 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.model.cart.Cart;
+import com.es.phoneshop.model.cart.CartService;
+import com.es.phoneshop.model.cart.HttpSessionCartService;
+import com.es.phoneshop.model.order.DefaultOrderService;
+import com.es.phoneshop.model.order.Order;
+import com.es.phoneshop.model.order.OrderService;
+import com.es.phoneshop.model.order.PaymentMethod;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CheckoutPageServlet extends HttpServlet {
+    private CartService cartService;
+    private OrderService orderService;
+
+    private static final String ORDER = "order";
+    private static final String FIRST_NAME = "firstName";
+    private static final String LAST_NAME = "lastName";
+    private static final String PHONE = "phone";
+    private static final String DELIVERY_DATE = "deliveryDate";
+    private static final String DELIVERY_ADDRESS = "deliveryAddress";
+    private static final String PAYMENT_METHOD = "paymentMethod";
+    private static final String ERROR_MAP = "errorMap";
+
+    @Override
+    public void init() {
+        cartService = HttpSessionCartService.getInstance();
+        orderService = DefaultOrderService.getInstance();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        Cart cart = cartService.getCart(request);
+        Order order = orderService.getOrder(cart);
+
+        request.setAttribute(ORDER, order);
+
+        request.getRequestDispatcher("/WEB-INF/pages/checkout.jsp").forward(request, response);
```

Comment:
Move strings (especially strings used in multiple places like "/WEB-INF/pages/checkout.jsp") to constants.